train/arch-and-scaling-template.slurm

```diff
@@ -177,5 +177,13 @@ export CMD=" \
 
 echo $CMD
 
+# We create the folder where the logs will be stored. This command assumes that only the last folder of the logs path
+# can be missing
+if [[ ! -d "${LOGS_PATH}" ]]
+then
+    echo "${LOGS_PATH} doesn't exists on your filesystem. We create it."
+    mkdir $LOGS_PATH
+fi
+
 # to debug - add echo (it exits and prints what it would have launched)
-srun --jobid $SLURM_JOBID bash -c '$LAUNCHER --node_rank $SLURM_PROCID $CMD' 2>&1 | tee $LOGS_PATH/tr3d-1B3-more-warmup.$SLURM_JOBID.out
+srun --jobid $SLURM_JOBID bash -c '$LAUNCHER --node_rank $SLURM_PROCID $CMD' 2>&1 | tee -a $LOGS_PATH/tr3d-1B3-more-warmup.$SLURM_JOBID.out
```

Comment:
why `-a`? We have already a unique log file with `.$SLURM_JOBID.out`

Reply:
but actually the change should be different. 

At least in the main training there is just one central log, so it's just:

```suggestion
srun --jobid $SLURM_JOBID bash -c '$LAUNCHER --node_rank $SLURM_PROCID $CMD' 2>&1 | tee -a $LOGS_PATH/main_log.txt
```

Reply:
see: https://github.com/bigscience-workshop/bigscience/blob/7458544256e13f370e638e70b30a52a444f9343a/train/tr8-104B-wide/tr8-104B.slurm#L169

Reply:
Oops, I thought I didn't push the commit with this change. Indeed, as it is, it was not very useful. Personally I'm happy to have only one log file too.